src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -99,68 +112,151 @@ const ChannelImage = ({
     return (
       <View>
-        {/* <FastImage source={{uri: mainPicture}} style={styles.image} /> */}
         {renderMainImage()}
         <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
       </View>
     );
+  }
 
-  // MY POST NOTIFICATION I COMMENTED CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED && isOwnSignedPost)
+  // POST NOTIFICATION IMAGE FOR SIGNED TAB
+  if (type === BaseChannelItemTypeProps.MY_SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY) {
     return (
       <View>
-        {/* Chat Image */}
         <FastImage source={{uri: mainPicture}} style={styles.image} />
-        {/* Post Notification Image */}
-        <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
+        <ChannelAnonymousSubImage
+          anonPostNotificationUserInfo={{
+            anon_user_info_emoji_code: anonPostNotificationUserInfo?.anon_user_info_emoji_code,
+            anon_user_info_color_code: anonPostNotificationUserInfo?.anon_user_info_color_code
+          }}
+        />
       </View>
     );
+  }
 
-  // MY POST NOTIFICATION I COMMENTED CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED)
+  if (
+    type === BaseChannelItemTypeProps.MY_SIGNED_POST_NOTIFICATION_I_COMMENTED ||
+    type === BaseChannelItemTypeProps.MY_SIGNED_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY ||
+    type === BaseChannelItemTypeProps.MY_SIGNED_POST_NOTIFICATION_COMMENTED ||
+    type === BaseChannelItemTypeProps.SIGNED_POST_NOTIFICATION_I_COMMENTED ||
+    type === BaseChannelItemTypeProps.SIGNED_POST_NOTIFICATION
+  ) {
     return (
       <View>
-        {/* Chat Image */}
-        <FastImage source={AnonymousProfile} style={styles.image} />
-        {/* Post Notification Image */}
-        <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
+        <FastImage source={{uri: mainPicture}} style={styles.image} />
+        {renderMyPostNotificationSubImage()}
       </View>
     );
+  }
 
-  // MY POST NOTIFICATION CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION)
+  if (type === BaseChannelItemTypeProps.MY_SIGNED_POST_NOTIFICATION) {
     return (
       <View>
-        {/* Chat Image */}
-        <FastImage source={AnonymousProfile} style={styles.image} />
-        {/* Post Notification Image */}
+        <FastImage source={{uri: mainPicture}} style={styles.image} />
+        <View style={[styles.postNotificationImage, styles.myPostNotificationImageContainer]}>
+          <FastImage source={FeedIcon} style={styles.postNotificationIcon} />
+        </View>
+      </View>
+    );
+  }
+
+  if (
+    type === BaseChannelItemTypeProps.ANON_POST_NOTIFICATION ||
+    type === BaseChannelItemTypeProps.ANON_POST_NOTIFICATION_I_COMMENTED
+  ) {
+    return (
+      <View>
+        {renderMainImage()}
         {renderMyPostNotificationSubImage()}
       </View>
     );
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for rendering different types of images based on the `type` has been significantly refactored. Ensure that these changes correctly render the expected images for each type and don't introduce any regressions.
```diff
-  // ANON POST NOTIFICATION CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.ANON_POST_NOTIFICATION)
-     return (
-       <View>
-        <FastImage source={{uri: mainPicture}} style={styles.image} />
-         {renderMyPostNotificationSubImage()}
-       </View>
-     );
- 
-  // ANON POST NOTIFICATION I COMMENTED CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.ANON_POST_NOTIFICATION_I_COMMENTED)
-     return (
-       <View>
-        {/* <FastImage source={{uri: mainPicture}} style={styles.image} /> */}
-         {renderMainImage()}
-         <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
-       </View>
-     );
- 
-  // MY POST NOTIFICATION I COMMENTED CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED && isOwnSignedPost)
-     return (
-       <View>
-        {/* Chat Image */}
-         <FastImage source={{uri: mainPicture}} style={styles.image} />
-        {/* Post Notification Image */}
-        <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
-       </View>
-     );
- 
-  // MY POST NOTIFICATION I COMMENTED CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED)
-     return (
-       <View>
-        {/* Chat Image */}
-        <FastImage source={AnonymousProfile} style={styles.image} />
-        {/* Post Notification Image */}
-        <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
-       </View>
-     );
- 
-  // MY POST NOTIFICATION CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION)
-     return (
-       <View>
-        {/* Chat Image */}
-        <FastImage source={AnonymousProfile} style={styles.image} />
-        {/* Post Notification Image */}
-         {renderMyPostNotificationSubImage()}
-       </View>
-     );
+  // POST NOTIFICATION IMAGE FOR ANONYMOUS TAB
+  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY) {
+     return (
+       <View>
+        <FastImage source={AnonymousProfile} style={styles.image} />
+        <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
+      </View>
+    );
+  }
+ 
+  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY) {
+    return (
+      <View>
+        <FastImage source={AnonymousProfile} style={styles.image} />
+        <ChannelAnonymousSubImage
+          anonPostNotificationUserInfo={{
+            anon_user_info_emoji_code: anonPostNotificationUserInfo?.anon_user_info_emoji_code,
+            anon_user_info_color_code: anonPostNotificationUserInfo?.anon_user_info_color_code
+          }}
+        />
+      </View>
+    );
+  }
+ 
+  if (
+    type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_I_COMMENTED ||
+    type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION_COMMENTED
+  ) {
+    return (
+      <View>
+        <FastImage source={AnonymousProfile} style={styles.image} />
+         {renderMyPostNotificationSubImage()}
+       </View>
+     );
+  }
+ 
+  if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION) {
+    return (
+      <View>
+        <FastImage source={AnonymousProfile} style={styles.image} />
+        <View style={[styles.postNotificationImage, styles.myPostNotificationImageContainer]}>
+          <FastImage source={FeedIcon} style={styles.postNotificationIcon} />
+        </View>
+      </View>
+    );
+  }
+ 
+  if (
+    type === BaseChannelItemTypeProps.ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY ||
+    type === BaseChannelItemTypeProps.SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY
+  ) {
+     return (
+       <View>
+         {renderMainImage()}
+         <FastImage source={AnonymousProfile} style={styles.postNotificationImage} />
+


<!-- This is an auto-generated comment by OSS CodeRabbit -->